domain/application/service/application.go

```diff
@@ -516,7 +535,7 @@ func (s *ApplicationService) RegisterCAASUnit(ctx context.Context, appName strin
 			return s.insertCAASUnit(ctx, appID, args.OrderedId, unitArg)
 		}
 		if unitLife == life.Dead {
-			return fmt.Errorf("dead unit %q already exists%w", args.UnitName, errors.Hide(applicationerrors.ApplicationIsDead))
+			return fmt.Errorf("dead unit %q already exists%w", args.UnitName, errors.Hide(applicationerrors.UnitAlreadyExists))
```

Comment:
Why it's UnitAlreadyExists now?

Reply:
It should have been "unit" originally. It's a single unit being operated on.